src/site/topic16.rst

```diff
@@ -134,6 +134,19 @@ Array Iterator
     * If we want to iterate over the collection again, we create a new iterator
 
 
+.. warning::
```

Comment:
Sphinx does have [notes](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html) too.